src/pages/team/index.tsx

```diff
@@ -41,35 +41,16 @@ const TeamComposition = ({teamData}: {teamData: TeamData}) => {
   }
 }
 
-type SubscriptionData = {
-  portalUrl: string | undefined
-}
-
-const AtCapacityNotice = ({teamData}: {teamData: TeamData}) => {
-  const {accountSlug, stripeCustomerId} = teamData
-  const [
-    subscriptionData,
-    setSubscriptionData,
-  ] = React.useState<SubscriptionData>({} as SubscriptionData)
-
-  React.useEffect(() => {
-    if (stripeCustomerId) {
-      axios
-        .get(`/api/stripe/billing/session`, {
-          params: {
-            customer_id: stripeCustomerId,
-            account_slug: accountSlug,
-          },
-        })
-        .then(({data}) => {
-          if (data) {
-            setSubscriptionData(data)
-          }
-        })
-    }
-  }, [stripeCustomerId, accountSlug])
```

Comment:
This was the duplicate Stripe API call that has been pulled up to a single call site.